packages/web/pages/learn/metaradio.tsx

```diff
@@ -0,0 +1,32 @@
+import { LoadingState } from '@metafam/ds';
+import { PageContainer } from 'components/Container';
+import { HeadComponent } from 'components/Seo';
+import React, { useState } from 'react';
+
+import { descriptions } from '../../utils/menuLinks';
+
+const ForumPage: React.FC = () => {
+  const [loading, setLoading] = useState(true);
+
+  return (
+    <PageContainer p={0} position="fixed">
+      {loading && <LoadingState position="absolute" />}
+      <HeadComponent
+        title={`MetaRadio`}
+        description={descriptions.metaradio}
+        url="https://anchor.fm/MetaGame/"
+      />
+      <iframe
```

Comment:
Would it make sense to have a shared component that just accepts a `title` and `src`? Otherwise they seem to be all the same

Reply:
I thought about it, but it implementing it with a reusable component wouldn't make a major difference and we would probably want to replace these iframes with content that's inside of mymeta anyway, so I'd keep it like that, because the extra work would not bring any visible benefit.